entgql/internal/todo/ent.graphql

```diff
@@ -3,151 +3,165 @@ CategoryWhereInput is used for filtering Category objects.
-  textEqualFold: String
-  textContainsFold: String
-  
-  """status field predicates"""
-  status: CategoryStatus
-  statusNEQ: CategoryStatus
-  statusIn: [CategoryStatus!]
-  statusNotIn: [CategoryStatus!]
-  
-  """config field predicates"""
-  config: CategoryConfigInput
-  configNEQ: CategoryConfigInput
-  configIn: [CategoryConfigInput!]
-  configNotIn: [CategoryConfigInput!]
-  configGT: CategoryConfigInput
-  configGTE: CategoryConfigInput
-  configLT: CategoryConfigInput
-  configLTE: CategoryConfigInput
-  configIsNil: Boolean
-  configNotNil: Boolean
-  
-  """duration field predicates"""
-  duration: Duration
-  durationNEQ: Duration
-  durationIn: [Duration!]
-  durationNotIn: [Duration!]
-  durationGT: Duration
-  durationGTE: Duration
-  durationLT: Duration
-  durationLTE: Duration
-  durationIsNil: Boolean
-  durationNotNil: Boolean
-  
-  """count field predicates"""
-  count: Uint64
-  countNEQ: Uint64
-  countIn: [Uint64!]
-  countNotIn: [Uint64!]
-  countGT: Uint64
-  countGTE: Uint64
-  countLT: Uint64
-  countLTE: Uint64
-  countIsNil: Boolean
-  countNotNil: Boolean
-  
-  """id field predicates"""
-  id: ID
-  idNEQ: ID
-  idIn: [ID!]
-  idNotIn: [ID!]
-  idGT: ID
-  idGTE: ID
-  idLT: ID
-  idLTE: ID
-  
-  """todos edge predicates"""
-  hasTodos: Boolean
-  hasTodosWith: [TodoWhereInput!]
+	not: CategoryWhereInput
+	and: [CategoryWhereInput!]
+	or: [CategoryWhereInput!]
+	"""
+	text field predicates
+	"""
+	text: String
+	textNEQ: String
+	textIn: [String!]
+	textNotIn: [String!]
+	textGT: String
+	textGTE: String
+	textLT: String
+	textLTE: String
+	textContains: String
+	textHasPrefix: String
+	textHasSuffix: String
+	textEqualFold: String
+	textContainsFold: String
+	"""
+	status field predicates
+	"""
+	status: CategoryStatus
+	statusNEQ: CategoryStatus
+	statusIn: [CategoryStatus!]
+	statusNotIn: [CategoryStatus!]
+	"""
+	config field predicates
+	"""
+	config: CategoryConfigInput
+	configNEQ: CategoryConfigInput
+	configIn: [CategoryConfigInput!]
+	configNotIn: [CategoryConfigInput!]
+	configGT: CategoryConfigInput
+	configGTE: CategoryConfigInput
+	configLT: CategoryConfigInput
+	configLTE: CategoryConfigInput
+	configIsNil: Boolean
+	configNotNil: Boolean
+	"""
+	duration field predicates
+	"""
+	duration: Duration
+	durationNEQ: Duration
+	durationIn: [Duration!]
+	durationNotIn: [Duration!]
+	durationGT: Duration
+	durationGTE: Duration
+	durationLT: Duration
+	durationLTE: Duration
+	durationIsNil: Boolean
+	durationNotNil: Boolean
+	"""
+	count field predicates
+	"""
+	count: Uint64
+	countNEQ: Uint64
+	countIn: [Uint64!]
+	countNotIn: [Uint64!]
+	countGT: Uint64
+	countGTE: Uint64
+	countLT: Uint64
+	countLTE: Uint64
+	countIsNil: Boolean
+	countNotNil: Boolean
+	"""
+	id field predicates
+	"""
+	id: ID
+	idNEQ: ID
+	idIn: [ID!]
+	idNotIn: [ID!]
+	idGT: ID
+	idGTE: ID
+	idLT: ID
+	idLTE: ID
+	"""
+	todos edge predicates
+	"""
+	hasTodos: Boolean
+	hasTodosWith: [TodoWhereInput!]
 }
```

Comment:
Can we control the printer to use 2 spaces instead of a tab? I think it's more common in GraphQL (and JS).

Reply:
Nope, there's no such option in the gqlparser's formatter.

Reply:
Sent PR: https://github.com/vektah/gqlparser/pull/208

Reply:
💯